src/test/kotlin/pt/up/fe/ni/website/backend/controller/EventControllerTest.kt

```diff
@@ -63,18 +86,29 @@ internal class EventControllerTest @Autowired constructor(
     @EndpointTest
     @DisplayName("POST /events/new")
     inner class CreateEvent {
+        @BeforeAll
+        fun addAccount() {
+            accountRepository.save(testAccount)
+        }
+
         @Test
         fun `should create a new event`() {
             mockMvc.post("/events/new") {
                 contentType = MediaType.APPLICATION_JSON
-                content = objectMapper.writeValueAsString(testEvent)
+                content = objectMapper.writeValueAsString(
+                    mapOf(
+                        "title" to testEvent.title,
+                        "description" to testEvent.description,
+                        "date" to testEvent.date,
+                        "teamMembersIds" to mutableListOf(testAccount.id!!),
+                    )
+                )
             }
                 .andExpect {
                     status { isOk() }
                     content { contentType(MediaType.APPLICATION_JSON) }
                     jsonPath("$.title") { value(testEvent.title) }
                     jsonPath("$.description") { value(testEvent.description) }
-                    jsonPath("$.date") { value(containsString("28-07-2022")) }
```

Comment:
Why did this disappear? Also, we should probably test if teamMembers are returned